app/helpers/comments_helper.rb

```diff
@@ -1,4 +1,7 @@
 module CommentsHelper
+  MAX_NBR_OF_COMMENTS_TO_RENDER = 250
+  MIN_NBR_OF_COMMENTS_TO_RENDER = 8
```

Comment:
**non-blocking suggestion**: I think these could just read `MAX_COMMENTS_TO_RENDER` and `MIN_COMMENTS_TO_RENDER`

Reply:
+1 to this suggestion!

Reply:
That is fine by me. I will change it.